wgengine/pendopen.go

```diff
@@ -122,6 +123,13 @@ func (e *userspaceEngine) trackOpenPostFilterOut(pp *packet.Parsed, t *tstun.Wra
 		}
 	}
 
+	// Accepted a flow, log the flow kind
+	if tsaddr.IsTailscaleIP(flow.Dst.IP()) {
```

Comment:
This is a different definition of "relayed". You're doing "using a subnet relay", but I thought what we wanted for stats was direct connects vs. DERP relays. The two are independent.

Reply:
Is there a convenient way to check for DERPs here? I assume I'd need the DERP map so I guess I need to move this somewhere else

Reply:
I've added a metric for derps in magicsock